packages/webapp/.env.example

```diff
@@ -5,4 +5,15 @@ NEXT_PUBLIC_BASESCAN_API_KEY=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID=XXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
We'll have some redundant env options to choose which subgraphs to use untill we settle on a final solution. I put all the values in prod vercel so any of these variables work as long as your code is pointing to the one you want